tests/cli/test_cli.py

```diff
@@ -425,7 +427,7 @@ def __exit__(self, exc_type, exc_value, tb):
                 _run_command([cl, 'rm', '--force', bundle])
 
         # Reset disk quota
-        _run_command([cl, 'uedit', 'codalab', '--disk-quota', '100g'])
+        _run_command([cl, 'uedit', 'codalab', '--disk-quota', self.disk_quota])
```

Comment:
We should initialize self.disk_quota to `None` in `__enter__`, then run this command only if `self.disk_quota is not None`

Reply:
do you mean initialize `self.disk_quota` to `None` in `__init__`?

Reply:
Yes